test/ExecutionFramework.h

```diff
@@ -241,6 +241,12 @@ class ExecutionFramework
 		return result;
 	}
 
+	util::h160 setAccount(size_t _accountNumber)
```

Comment:
Hmm, how about` `changeCurrentAccount` or `setCurrentSender` or something in that regard?